hyperdome/client/hyperdome_client.py

```diff
@@ -242,8 +239,13 @@ def handle_error(self, error: Exception):
         changes active window with new text
         and brings to focus if currently in the background.
         """
-        self.error_window.setText(error.args[0] or "no error description provided")
-        self.__log.debug(f'Received "{type(error)}" from task')
+        self.error_window.setText(
+            error.args[0]
+            if isinstance(error.args[0], str)
+            else "no error description provided"
+        )
+        self.__log.debug(f"Received error from task, set logging to TRACE for info")
```

Comment:
I'm surprised Black didn't complain that you're using an fstring with no variables

Reply:
huh, I wrote that so long ago. I wonder what I even made it an f-string.
I'm also surprised you're not complaining that I'm using an fstring with no variables :P